tensorrt_llm/_torch/pyexecutor/kv_cache_transceiver.py

```
@@ -5,6 +5,7 @@
 from tensorrt_llm import logger
 from tensorrt_llm.bindings import WorldConfig
 from tensorrt_llm.bindings.executor import CacheTransceiverConfig
+from tensorrt_llm.logger import logger
```

Ruff (0.12.2)

Redefinition of unused `logger` from line 5

(F811)